### PR TITLE
feat(home): Add contributors sectioin

### LINKED ISSUE #51: FEATURE - Add github contributors to the home page, below sponsors/financial backers
Related to notes here for financial contributors on opencollective https://github.com/TampaDevs/tampadevs/issues/50

We should also add a list of github contributors on the homepage who help build out new features

This will be right below all the sponsors and financial contributors. Here's  an example mockup image from React-Hook-Form

image

Not 100% sure how to pull up the list of github contributors, but github should have some tools for doing this

### PR DESCRIPTION
**Pull request by RyanClementsHax**
- Add section showcasing contributors section
- Realign supporter sections to be evenly spaced
- Format home.njk
- Remove cta button background in financial backers section to avoid clashing with visual hierarchy
- Ensure consistent spacing between cta buttons and the sections they're a cta for

Fixes #51

### CODE REVIEW

**Comment by RyanClementsHax** [comment]
@chtzvt Lmk if this is what you're looking for

File: src/_includes/layouts/home.njk [hunk 1]
```diff
@@ -54,20 +56,35 @@ layout: layouts/base.njk
...
       <a class="sponsor-button td-alert-banner" href="https://go.tampa.dev/donate">Donate</a>
     </div>
-  </div>
-</section>
+  </section>
+  <section class="container sponsor-container">
+    {# Financial Backers #}
+    <h3>Financial Backers</h3>
+    <div class="sponsor-list sponsor-list__silver">
+      <a class="financial-backers-cta" href="https://opencollective.com/tampadevs#backers">
```
Comment (manuelosorio) [inline src/_includes/layouts/home.njk]: _Financial Backers_ section can be removed since the _Contributing Members_ section already shows the donators.

**Review by manuelosorio — CHANGES_REQUESTED** [review]
Only 1 change is needed. Aside from that I think you did a good job with this pr.

File: src/_includes/layouts/home.njk (hunk 1 above)
Comment (RyanClementsHax) [reply src/_includes/layouts/home.njk]: Thanks. I updated it

**Review by manuelosorio — APPROVED** [review]
LGTM!

**Pull request merged**